frontend/src/components/users/Wishlist.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import axios from 'axios';
+
+
+
+const Wishlist = (props) => {
+	return (
+		<div class="card col-md-6 ">
```

Comment:
Typo : all html tags of this components needs to correction on class (className)